### PR TITLE
add hdfs_doc_mapper.json

### PR DESCRIPTION
**Pull request by PSeitz**

### CODE REVIEW

**Comment by fulmicoton** [comment]
I recommend renaming `hdfslogsdataset_doc_mapper.json` or something like that to make sure people don't think it is a conf to use hdfs as a storage.

File: examples/doc_mappers/hdfs_doc_mapper.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "store_source": true,
```
Comment (fulmicoton) [inline examples/doc_mappers/hdfs_doc_mapper.json:2]: I think `"store_source": false` might be better here. (The json in the _source field can be confusing)

**Comment by PSeitz** [comment]
> I recommend renaming `hdfslogs_doc_mapper.json` or something like that to make sure people don't think it is a conf to use hdfs as a storage.

ah yeah true

File: examples/doc_mappers/hdfs_doc_mapper.json (hunk 1 above)
Comment (PSeitz) [reply examples/doc_mappers/hdfs_doc_mapper.json]: done

**Pull request merged**